### PR TITLE
Enhance project structure with cookiecutter

### LINKED ISSUE #12: Don't initialize app in __init__.py
Use the pattern used in this project: https://github.com/cookiecutter-flask/cookiecutter-flask

### PR DESCRIPTION
**Pull request by Haider8**
fixes #12

### CODE REVIEW

**Comment by Haider8** [comment]
@iamareebjamal What are the main things which I should remove?

**Comment by iamareebjamal** [comment]
Everything related to webpack and templates

**Comment by iamareebjamal** [comment]
https://www.thefreedictionary.com/improvise

File: package.json [hunk 1]
```diff
@@ -3,9 +3,8 @@
   "version": "1.0.0",
```
Comment (iamareebjamal) [inline package.json]: The whole file isn't required

Comment (Haider8) [reply package.json]: As well as `package-lock`

Comment (iamareebjamal) [reply package.json]: yes

**Comment by Haider8** [comment]
@iamareebjamal Should I also remove all the `npm` commands?

**Comment by iamareebjamal** [comment]
Yes

**Comment by Haider8** [comment]
@iamareebjamal I have updated the PR. There are still some things which aren't needed like bcrypt and flask-caching, so should I remove all those? Secondly, the tests are to be written for the new code. Should I open new PR for that or do it here only? I have one doubt also, is the current way to list the requirements is right as far as detection by dependabot is concerned?

**Comment by iamareebjamal** [comment]
Flask caching and bcrypt are fine. Tests should be done in a separate PR

Yes, requirements are good

**Pull request merged**